docs/database.md

```diff
@@ -294,3 +294,120 @@ def create_test_database():
 
 [sqlalchemy-core]: https://docs.sqlalchemy.org/en/latest/core/
 [alembic]: https://alembic.sqlalchemy.org/en/latest/
+
+
+## Integrating `SQLAlchemy` ORM
```

Comment:
My 2 cents:
The existing example is already using sqlalchemy (but in an async way with `databases`), this looks more like a migration example, maybe the title could be renamed to `Migrating from Flask`. What do you think?

Reply:
The existing example uses SQLAlchemy core with async. This uses SQLAlchemy ORM with async.

The focus is on being able to use the ORM too.